src/lucky_record/criteria.cr

```diff
@@ -1,7 +1,9 @@
 class LuckyRecord::Criteria(T, V)
   property :rows, :column
+  @negate_next_criteria : Bool
```

Comment:
You could add ` = false` to the end of this and I think you wouldn't need the instance var on line 6

Reply:
I thought so as well, but we get a `this 'initialize' doesn't initialize instance variable '@negate_next_criteria'`

Reply:
👍